relay-server/src/actors/upstream.rs

```diff
@@ -223,14 +245,27 @@ struct UpstreamRequest {
 }
 
 pub struct UpstreamRelay {
+    /// backoff policy for the registration messages
     backoff: RetryBackoff,
+    /// backoff policy for the network outage message
```

Comment:
is there such a thing as a network outage message?

Reply:
Yes, it is the `CheckUpstreamConnection` message :-). I went back and forth a few times with the name and I'm still not very happy with the result.